packages/ng/core-select/input/select-input.component.ts

```diff
@@ -295,7 +295,7 @@ export abstract class ALuSelectInputComponent<TOption, TValue> implements OnDest
 			this.clueChanged(clue);
 			this._panelRef = this.buildPanelRef();
 			this.bindInputToPanelRefEvents();
-		});
+		}, 10);
```

Comment:
Does it work without this ? 0 and 10 should approximatively do the same and I would prefer not counting on that 😅

Reply:
At least it passes manual tests.
Should we consider another option?

Reply:
When it was 0, we had a problem.

Reply:
The issue is that we need the DOM to be updated before placing the panel, as it can add a line for the input now that it's shown. 0 is not enough because it relies on zone for that, 10 is enough for the DOm update to happen and place the panel properly.

Reply:
could we use the new afterNextRender?